app/models/export_template.rb

```diff
@@ -19,6 +20,90 @@ def set_default_values
     end
   end
 
+  def zip?
+    kind == 'zip'
+  end
+
+  def tabular?
+    kind != 'zip'
+  end
+
+  def paths=(full_paths)
+    content["columns"] = full_paths.compact_blank
+      .map { [_1, full_path_hash(_1)] }
+      .filter { |_, current_column| current_column.present? }.map do |full_path, current_column|
```

Comment:
- `full_path` est uniquement utilisé dans la méthode `libelle_for_path`. Cette dernière devrait prendre en argument un hash (je soupsonne l'implem actuelle d'être buggué)

- Par ailleurs full_path_hash peut être inliner directement ici par JSON.parse

- finalement on a pas besoin de refilter current_column si on a deja fait un compact_blank initialement.

du coup, tada :

```suggestion
  def paths=(json_paths)
    content["columns"] = json_paths.compact_blank.map { JSON.parse(_1) } do |path_hash|
```